pkg/noderesourcetopology/filter.go

```diff
@@ -41,10 +41,16 @@ func singleNUMAContainerLevelHandler(pod *v1.Pod, zones topologyv1alpha1.ZoneLis
 	nodes := createNUMANodeList(zones)
 	qos := v1qos.GetPodQOS(pod)
 
+	logNumaNodes("cnt handler NUMA resources", nodes)
```

Comment:
This should be logged out to the user, so I would suggest not spare few letter and user ```container``` and not ```cnt```

Reply:
good point again. I used `cnt` because it is exactly long as `pod`, but a bit more verbosity helps here.